docs/formats.md

```diff
@@ -33,8 +33,7 @@ There is an extensive (but not exhaustive) list of [included formats](#pre-defin
 
 ### Format configuration
 
-Formats can take configuration to make them more flexible. This allows you to re-use the same format multiple times with different configuration or to allow the format to use data not defined in the tokens themselves. To configure a format add extra attributes on the file object in your configuration like so:
-
+Formats can take configuration to make them more flexible. This allows you to re-use the same format multiple times with different configurations or to allow the format to use data not defined in the tokens themselves. To configure a format, add extra attributes on the file object in your configuration like the following:
```

Comment:
Did you make these changes in **scripts/handlebars/templates/formats.hbs** and run `npm run generate-docs`? If not these changes will get overridden when we deploy the documentation site.

Reply:
Please do; it will overwrite these files. This will help us understand if any changes need to be made.

Reply:
@dbanksdesign @chazzmoney Can you help me, what changes are need to be done?

Reply:
@dbanksdesign @chazzmoney I did these changes in template files. Can you verify the new changes?